src/src_/rpc.toit

```diff
@@ -22,10 +22,16 @@ abstract class RpcService:
   static PROPERTIES_ ::= ble.CHARACTERISTIC-PROPERTY-READ | ble.CHARACTERISTIC-PROPERTY-WRITE
   static PERMISSIONS_ ::= ble.CHARACTERISTIC-PERMISSION-READ | ble.CHARACTERISTIC-PERMISSION-WRITE
 
+  static hash-counter_ := 0
+
+  // When a new RPC call comes in, we clear all other RPC services' return values.
+  static all-rpc-services_/Set? ::= {}
+
   characteristic_/ble.LocalCharacteristic
   description/string
   security_/Security?
   task_/Task? := null
+  hash-code/int ::= hash-counter_++
```

Comment:
I could be convinced that getting a random smi here would work well and you would avoid the counter.

Reply:
done.